### PR TITLE
feat: add create employee component

### PR DESCRIPTION
**Pull request by punwai**
## Description
- Add create employee component

## Related issues and discussion
#157

## Screenshots, if any
Screenshot 2019-12-28 at 02 21 02

ezgif com-video-to-gif (1)

testval

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/organization/employees/create-employee/create-employee.component.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  /**
+   * Creates the employee form.
+   */
+  createEmployeeForm() {
+    this.employeeForm = this.formBuilder.group({
+      'officeId': ['', Validators.required],
+      'firstname': ['', Validators.required],
+      'lastname': ['', Validators.required],
+      'isLoanOfficer': [false, Validators.required],
```
Comment (abhaychawla) [inline src/app/organization/employees/create-employee/create-employee.component.ts]:
```suggestion
      'isLoanOfficer': [false],
```
Validator not required for this.

File: src/app/organization/employees/create-employee/create-employee.component.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  }
+
+  /**
+   * Creates the employee form.
+   */
+  createEmployeeForm() {
+    this.employeeForm = this.formBuilder.group({
+      'officeId': ['', Validators.required],
+      'firstname': ['', Validators.required],
+      'lastname': ['', Validators.required],
```
Comment (abhaychawla) [inline src/app/organization/employees/create-employee/create-employee.component.ts]: Refer to the community app's code. Another validation is supposed to be put here. Update accordingly.

Comment (punwai) [reply src/app/organization/employees/create-employee/create-employee.component.ts]: Added the implemented validator from the community app (First letter must be a A-z, the second letter can be anything). 

The regex in the community app is different, but it seems like angularJS treats regex slightly differently than here. The regex in the community app actually matches every character for the second letter, but the code is supposed to match A-z, . , - and '. 

When I applied the same regex to here it invalidated all special characters apart from .,- and numbers, so I implemented this instead to match the actual behaviour in the community app.

**Comment by punwai** [comment]
Made changes, and changed the commit message. Please have a look! @Jov03 @abhaychawla

**Comment by punwai** [comment]
@abhaychawla @Jov03 Rebased and ready to merge.

**Comment by abhaychawla** [comment]
@punwai As you are already aware and able to resolve merge conflicts and rebase on your own I would request you to please wait, let us help the other students first. We will merge this in the end. Is it okay?

**Comment by punwai** [comment]
@abhaychawla Of course, it is absolutely alright. I just thought that I'd rebase it incase you want to merge the PR, but if you prefer to do it later then that's completely fine by me.

**Comment by punwai** [comment]
@abhaychawla Will there be more web app tasks coming out??

**Comment by abhaychawla** [comment]
Yes, we will have more tasks soon. You can resolve the conflicts now.

**Comment by punwai** [comment]
@abhaychawla Conflicts are resolved.

**Pull request merged**